dev-site.yml

```diff
@@ -24,7 +24,7 @@ asciidoc:
     - ./lib/tab-block.js
 ui:
   bundle:
-    url: ../rhd-tutorial-ui/build/ui-bundle.zip
+    url: https://github.com/redhat-developer-demos/rhd-tutorial-ui/releases/download/v0.0.4/ui-bundle.zip
```

Comment:
I think that in most cases you work in dev mode to render the local changes, but you might relay on a published UI, so it always works (if you don't have the theme downloaded locally it throws an exception).